docs/integrations/datadog.md

```diff
@@ -21,6 +21,19 @@ If you're using Aiven for Apache Kafka® you can also
 [customise the metrics sent to Datadog](/docs/products/kafka/howto/datadog-customised-metrics).
 :::
 
+:::note
+Datadog integration is not available for new Startup-2 plans in Aiven for Apache Kafka.
+Existing customers using Startup-2 with Datadog integration can continue to create new
+Startup-2 plans with Datadog integration and use their existing services without
+upgrading to a higher plan.
```

Comment:
Consider replacing "customers" with addressing them directly using "you":

```suggestion
If you have already used Startup-2 with Datadog integration, you can continue to create new
Startup-2 plans with Datadog integrations and use your existing services without
upgrading to a higher plan.
```

Reply:
I used 'Existing customers' instead of 'you' to maintain a more neutral and formal tone, as this applies to a specific group of users rather than addressing all readers directly. This also keeps consistency with other similar documentation.